_posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc

```diff
@@ -1296,37 +1341,36 @@ This is a python script that does the following:
   `stdout` and `stderr`.
 - Exit with the appropriate exit code depending on if the underlying command succeeded or failed.
 
-Create a placeholder module called `ecs-deploy-runner` in `infrastructure-modules`, with a folder `docker` with the
-`Dockerfile` for creating the Docker image and the `known_hosts` file. Copy over the `Dockerfile` and `known_hosts` file
-from https://github.com/gruntwork-io/module-ci/tree/master/modules/ecs-deploy-runner/docker[module-ci]:
+You can use any Docker image with the ECS Deploy Runner, but it must meet the following requirements:
 
-----
-infrastructure-modules
-  └ cicd
-    └ ecs-deploy-runner
-      └ docker
-        └ Dockerfile
-        └ known_hosts
-    └ ecr-repo
-      └ main.tf
-      └ outputs.tf
-      └ variables.tf
-  └ networking
-    └ vpc-mgmt
-      └ main.tf
-      └ outputs.tf
-      └ variables.tf
-----
+- Define a trigger directory where scripts that can be invoked by the ECS Deploy Runner Lambda function are contained.
+  You want to limit the actions that can be performed within the ECS tasks with powerful IAM permissions. The ECS Deploy
```

Comment:
```suggestion
  You should limit the actions that can be performed within the ECS tasks due to its powerful IAM permissions. The ECS Deploy
```

I'd suggest a slight rephrasing here as it could be confusing which actions you're referring to - IAM "actions" or "script actions"

Reply:
Makes sense! Updated to clarify this is referring to scripts, and also expanded a bit why you want to limit the scripts more: e6300bf7